### ISSUE TITLE
stop working 

### ISSUE THREAD

**Issue by skarpinis** [Tommixoft/plugin.video.lostfilm.tv#2]
your script not working anymore 10:38:24.456 T:10876   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.KeyError'>
                                            Error Contents: 'data'
                                            Traceback (most recent call last):

**Comment by skarpinis** [comment]
10:38:24.493 T:3480   ERROR: XFILE::CDirectory::GetDirectory - Error getting plugin://plugin.video.lostfilm.tv/browse_series_episodes/254/Narcos
10:38:24.494 T:3480   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.lostfilm.tv/browse_series_episodes/254/Narcos) failed
10:38:24.650 T:11188  NOTICE: [xbmcswift2] Request for "/allseries" matches rule for function "allseries"

**Comment by Tommixoft** [comment]
Yeah LostFilm changed something again and plugin no longer works. Will try to fix it.

**Comment by Tommixoft** [comment]
O čia lietuvis, tia galiu lt rašyti :D Žodžiu vieną bėda isprendžiau, kitos nepavyksta... kažką rimto pakeitė..kolkas nematau ką.

**Comment by Tommixoft** [comment]
Looks like we're fucked. LostFilm.tv added capcha into login form, and i can not login. Looks like LostFilm plugin for now will be dead.

**Comment by Tommixoft** [comment]
FOUND HACK. You have to go to lostfilm.tv and login manually. Then lostfilm will not ask for capcha. And then KODI plugin can login without capcha

**Comment by Tommixoft** [comment]
LostFilm.tv added capcha protection. You MUST go to their website and login manually, then your device will eb able to login withouth capcha. You have to do this from 1 IP. Because LostFilm.tv remembers capcha settings based on IP. **This hack works 100%.**

**Comment by skarpinis** [comment]
yes working thanks. Dekui

**Comment by skarpinis** [comment]
svx, dekuj uz informacija norejau paklausti kodel kai pakeiciu 1080
nustatimuose video bet vistiek manes klausia kokia kokybe ziureti? ir
idejau puslapi serialia i favorites bet nerodo kodi programoje

2017 m. spalio 22 d. 18:32, Tomas <notifications@github.com> rašė:

**Comment by Tommixoft** [comment]
Deja, del lostfilmo logino apsaugos panasu, kad programa jau nebeveiks kaip veike... Nes programa pastoviai loginasi i lostfilm'a ir lostfilmas kai loginiesi praso ivesti skaicius kurie matomi paveiksliuke, o programiskai to padaryti neimanoma.
Kai nerodo favoritu-vadinasi programa nebegali prisiloginti ir reikia eiti vel per pc ir vel logintis rankiniu budu.

**Comment by skarpinis** [comment]
aisku ka padarysi seis laikais viskas komplikuotai :)

**Comment by Tommixoft** [comment]
Bandysiu zeti koda, man rodos scriptas logina daro su kiekviena uzklausa, kas gali papildomai issaukti kad Lostfilm praso ivesti paveiskliuke esanti koda. Bet kolkas esu labai uzimtas tai nera laiko paziureti.

**Comment by Tommixoft** [comment]
You can try 0.2.3 version https://github.com/Tommixoft/plugin.video.lostfilm.tv/releases

**Comment by iddqdidkfa** [comment]
`15:15:03.776 T:2872  NOTICE: [lostfilm.network_request] Fetching URL http://www.lostfilm.tv/v_search.php?c=366&s=1&e=1 with params None started...

15:15:03.882 T:2872  NOTICE: [lostfilm.network_request] Fetching URL http://www.lostfilm.tv/v_search.php?c=366&s=1&e=1 with params None taken: 0.106393 seconds

15:15:03.883 T:2872  NOTICE: [lostfilm.network_request] Fetching URL None with params None started...

15:15:03.884 T:2872  NOTICE: [lostfilm.network_request] Fetching URL None with params None taken: 0.000725 seconds

LocalizedError: Can't fetch URL: None (LostFilm.TV)

MissingSchema: Invalid URL 'None': No schema supplied. Perhaps you meant http://None?`

**Comment by Tommixoft** [comment]
@iddqdidkfa  Can not test until monday. But from log looks like it dont get URL from some function(s).
Try to login to lostfilm via web browser. then use KODI.

**Comment by iddqdidkfa** [comment]
@Tommixoft I did, that doesn't  help.

**Comment by Tommixoft** [comment]
@iddqdidkfa Asked one person to test his LostFilm plugin with Android Kodi, said all working ok.
I will be able to test only monday.
SO it doesn't work at all? Can you see list of tv shows? can you see episodes? favorites? or nothing works just main menu?

**Comment by iddqdidkfa** [comment]
@Tommixoft I have relogined on another browser and now it shows my favorites but another error comes:
```
Error Type: <type 'exceptions.ImportError'>
Error Contents: No module named torrent2http
```
I guess it's because I have installed torrent2http from ZIP file, not from repository. But install from repository failed many times and SuperRepo site works only after 10-15 refreshes. I know this problem is not with your plugin, but maybe you know where can I find correct torrent2http?
```
---
UPDATE: ok I found version 0.2.8 and now it all works like a charm, thanks.
---
Suggestion: maybe you can add some human readable notification if there are some login problems?

**Comment by Tommixoft** [comment]
@iddqdidkfa  SO all ok as i understand yes? I tested on windows, android all working so i think you also saying it's not ok.
Plugin written in Python and using Kodi framework which don't have much API documentation and i dont know Python.  In fact the only reason for me to try and fix this plugin to be working -is because of my mom using it, so i try to make sure it's working :D

torrent2htt can be download from here https://github.com/Tommixoft/script.module.torrent2http/releases/

**Comment by iddqdidkfa** [comment]
@Tommixoft yes, all working. It's not easy to install it from clear environment, but everything is ok now. Thanks.
